src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -4836,70 +4931,15 @@ public sealed class EnablePSRemotingCommand : PSCmdlet
-            #
-            # Windows PowerShell:
-            #   Microsoft.PowerShell
-            #   Microsoft.PowerShell32 (wow64)
-            #
-            # PowerShell Core:
-            #   PowerShell.<version ID>
-            #
-            $errorCount = $error.Count
-            $endPoint = Get-PSSessionConfiguration {0} -Force:$Force -ErrorAction silentlycontinue 2>&1
-            $newErrorCount = $error.Count
-
-            # remove the 'No Session Configuration matches criteria' errors
-            for ($index = 0; $index -lt ($newErrorCount - $errorCount); $index ++)
-            {{
-                $error.RemoveAt(0)
-            }}
-
-            $qMessage = $queryForRegisterDefault -f ""{0}"",""Register-PSSessionConfiguration {0} -force""
-            if ((!$endpoint) -and
-                ($force  -or $pscmdlet.ShouldProcess($qMessage, $captionForRegisterDefault)))
-            {{
-                $resolvedPluginInstallPath = """"
-                #
-                # Section 1:
-                # Move pwrshplugin.dll from $PSHOME to the endpoint directory
-                #
-                $pluginInstallPath = Join-Path ""$env:WINDIR\System32\PowerShell"" $psversiontable.GitCommitId
-                if (!(Test-Path $pluginInstallPath))
-                {{
-                    $resolvedPluginInstallPath = New-Item -Type Directory -Path $pluginInstallPath
-                }}
-                else
-                {{
-                    $resolvedPluginInstallPath = Resolve-Path $pluginInstallPath
-                }}
-                if (!(Test-Path $resolvedPluginInstallPath\{5}))
-                {{
-                    Copy-Item $PSHOME\{5} $resolvedPluginInstallPath -Force
-                    if (!(Test-Path $resolvedPluginInstallPath\{5}))
-                    {{
-                        Write-Error ($errorMsgUnableToInstallPlugin -f ""{5}"", $resolvedPluginInstallPath)
-                        return
-                    }}
-                }}
+            Register-EndpointIfNotPresent -Name {0} $Force $queryForRegisterDefault $captionForRegisterDefault
 
-                #
-                # Section 2:
-                # Generate the Plugin Configuration File
-                #
-                Generate-PluginConfigFile $resolvedPluginInstallPath
-
-                #
-                # Section 3:
-                # Register the endpoint
-                #
-                $null = Register-PSSessionConfiguration -Name {0} -force
-
-                set-item -WarningAction SilentlyContinue wsman:\localhost\plugin\{0}\Quotas\MaxShellsPerUser -value ""25"" -confirm:$false
-                set-item -WarningAction SilentlyContinue wsman:\localhost\plugin\{0}\Quotas\MaxIdleTimeoutms -value {4} -confirm:$false
-                restart-service winrm -confirm:$false
+            # Create the default PSSession configuration, not tied to any specific PowerShell version.
```

Comment:
Again, please add an example name in the comment to illustrate how the endpoint name should look.  I think this will help other developers reading the code.

Reply:
Comment added.